src/app/posts-module/posts-module.module.ts

```diff
@@ -1,18 +1,24 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { PostsComponent } from "./posts/posts.component";
+import { RouterModule } from "@angular/router";
+import { AppRoutingModule } from "../app-routing.module";
+import {PostComponent} from "./post/post.component";
 
 
 
 @NgModule({
   declarations: [
-    PostsComponent
+    PostsComponent,
+    PostComponent
   ],
   exports: [
     PostsComponent
   ],
-  imports: [
-    CommonModule
-  ]
+	imports: [
+		CommonModule,
+		RouterModule,
```

Comment:
Your `AppRoutingModule` already exports `RouterModule`. So, you don't need to import it